apps/mudita-center-e2e/src/helpers/add-device.helper.ts

```diff
@@ -0,0 +1,18 @@
+import { E2EMockClient } from "../../../../libs/e2e-mock/client/src"
+import { overviewDataWithOneSimCard } from "../../../../libs/e2e-mock/responses/src"
+
+export const addDeviceWithGivenSerialNumber = (serialNumber: string) => {
```

Comment:
function is not in use, so please remove it

Reply:
removed